examples/negate.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// How might I add my own support for --no-foo?
+
+// 1. const { parseArgs } = require('node:util'); // from node
+// 2. const { parseArgs } = require('@pkgjs/parseargs'); // from package
+const { parseArgs } = require('..'); // in repo
+
+const { values, tokens } = parseArgs({ strict: false, tokens: true });
```

Comment:
If we are going to land this example (which I think we should, either in this PR or a later one), I think this should probably use explicit options instead of `strict: false`, since it makes it clear that you need to specify a `'no-foo'` option explicitly when using `strict: true`. As is that's not obvious.

And at that point it might as well use `strict: true`, so you don't have to worry about stuff like `--foo=false` or `--no-foo=false`. It's possible to handle those, but it complicates the example more than necessary, I think.

Reply:
> If we are going to land this example (which I think we should, either in this PR or a later one)

I am keen to land some of these examples, but want to allow people to focus on the API and ignore reviewing the examples until a future PR if they wish. But comments are welcome now too.

> since it makes it clear that you need to specify a `'no-foo'` option explicitly when using `strict: true`, As is that's not obvious.

Oh, good point. I will change to strict and rework.

Reply:
Reworked with `strict: true` per suggestion